autopilot/sagemaker_autopilot_abalone_parquet_input.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
<p>I agree with @iaroslav-ai that the first line can be reworded for clarity. For minimal changes, how about: <code>This notebook exhibits the use of a Parquet dataset with SageMaker Autopilot.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3093/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done. Thank you</p>